SGMC/00_SRA_Cloud_metadata.ipynb

```diff
@@ -74,12 +78,31 @@
    "execution_count": 3,
    "id": "e54ec63d-fc1b-45e2-b1fb-2636a0123965",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Sorry, I realized I left the outputs in that shows my JSON key...  The outputs shouldn't have anything in it. 
```suggestion
   "outputs": [],
```